Toolkit/Public/Get-RscNasSystem.ps1

```diff
@@ -23,33 +26,57 @@ function Get-RscNasSystem {
 
     .EXAMPLE
     Get details of NAS system with id="72859a28-6276-555a-9a66-d93fe99d2751"
+    Get-RscNasSystem -Id "72859a28-6276-555a-9a66-d93fe99d2751"
+        Or 
     Get-RscNasSystem "72859a28-6276-555a-9a66-d93fe99d2751"
 
     .EXAMPLE
-    Retrieve list of NAS systems.
+    Retrieve list of NAS systems (first 2).
     Get-RscNasSystem -First 2
+    
+    .EXAMPLE
+    Retrieve all NAS systems. 
+    It will return at max 50 items (default page size).
```

Comment:
Pagination is built into the SDK, so it will return all NAS systems by automatically paging through.

Reply:
Remove explicit mentioning of "It will return at max 50 items (default page size)". Assuming customer might be aware of it already.